amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/settings/SettingsUtils.java

```diff
@@ -434,6 +436,15 @@ public static JsonBean getGeneralSettings() {
         return settings;
     }
 
+    private static String  getDefaultCurrencyCode() {
+        //It should never be null since its coming from gs but just in case we default to USD
+        AmpCurrency curr = CurrencyUtil.getEffectiveCurrency();
+        if (curr != null) {
+            return curr.getCurrencyCode();
+        } else {
+            return CurrencyUtil.BASE_CODE;
+        }
+    }
```

Comment:
Should we name it `getEffectiveCurrencyCode` for clarity that it may be different from the default one from WS? Please move this to `CurrencyUtil` for reuse.